plugins/remove-in-allns.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: remove-in-allns
```

Comment:
this is not a great name IMO. doesn't quite explain what's going on.

Reply:
I had thought about these names:

- remove-resource-everywhere
- remove-resource-allnamespaces

what do you think about one of these. Also if you have anything specific in mind, would be great to hear that.